spec/suites/layer/marker/MarkerSpec.js

```diff
@@ -159,7 +159,7 @@ describe("Marker", function () {
 			} else {
 				expect(marker._icon.src).to.contain('marker-icon.png');
 			}
-			expect(marker._icon.parentNode).to.be(map._panes.markerPane);
+			expect(marker._icon.parentNode.parentNode).to.be(map._panes.markerPane);
```

Comment:
Parent of parent is an overkill.
Maybe something wrong in marker design.

Reply:
I agree with you, it such a stupid thing I did.